crates/tauri/src/webview/mod.rs

```diff
@@ -132,6 +132,8 @@ pub struct InvokeRequest {
   pub headers: HeaderMap,
   /// The invoke key. Must match what was passed to the app manager.
   pub invoke_key: String,
+  /// The label of the webview that requested this command.
+  pub label: String,
```

Comment:
if we are gonna merge this PR as temporary fix, we should avoid adding this field.